PreloadedPersistentContainer.xcodeproj/project.pbxproj

```diff
@@ -699,6 +701,7 @@
 				PRODUCT_NAME = PreloadedPersistentContainer;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
```

Comment:
For reference: Since Carthage uses Release build and originally it optimizes the output, `libswiftSwiftOnoneSupport.dylib` is removed. But it is required to run the CLI (maybe only in Debug?). So I removed the optimization.